src/presentation/jsend-response-formatter.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpResponseFormatter } from './interfaces/http-response-formatter'
+import { ErrorCode } from '../model/enums/error-code'
+
+export class JsendResponseFormatter implements HttpResponseFormatter{
+  public formatErrorResponse(message: string, errorCode: ErrorCode): object {
+    return {
+      status: 'error',
+      code: errorCode,
+      message: message,
+    }
+  }
+
+  public formatFailResponse(data?: object): object {
+    return {
+      status: 'fail',
+      data: data ? data : null
```

Comment:
Haven't confirmed it, but this `data ?? null` should give the same result

Reply:
It should indeed, so i have updated the code to use it.